### ISSUE TITLE
Incompatible with ReplayMod (Crash on boot)

### ISSUE THREAD

**Issue by HRudyPlayZ** [FalsePattern/GasStation#10]
Everything's in the title.

**Comment by makamys** [comment]
That's because ReplayMod (1.7.10-2.5.2) uses methods that were removed in Mixin 0.8 in its bootstrap code. If you make these changes in ReplayModTweaker, it will work again (but not with Mixin 0.7)
```patch
-         if (GlobalProperties.get("mixin.initialised") == null) {
+         if (GlobalProperties.get(Keys.INIT) == null) {
```
```patch
-      platform.addContainer(uri);
+      platform.addContainer(new ContainerHandleURI(uri));
```
Here's a class with these changes you can drop in the jar (replacing `com/replaymod/core/tweaker/ReplayModTweaker.class`).

[ReplayModTweaker.class.zip](https://github.com/FalsePattern/GasStation/files/10096529/ReplayModTweaker.class.zip)


The issue could be solved on GasMix's side by readding those two methods. I don't know if backward compatibility with Mixin 0.7 is a goal though.

**Comment by FalsePattern** [comment]
Fixed in 0.5.0
